### PR TITLE
Pr/94

### PR DESCRIPTION
**Pull request by ColeFeely6**
Adding:

Giving corporate jurisdiction to the google analytics script

'Model Page'

The advanced edits for Tim

Fixing the image search issues

### CODE REVIEW

File: docs/2/technical/lorawan/lorawan-devices.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ toc: true
 ##### SUMMARY : 
 
 1. [Introduction](./#1-introduction)
-2. [Types of Sensors](./#2-types of sensors)
-3. [Sensors Microshare® Uses](./#3-sensors-microshare-uses)
-- A. [Browan Sensors](./#a-browan-sensors)
+2. [Types of Devices](./#2-types of devices)
+3. [DEvices Microshare® Uses](./#3-devices-microshare-uses)
```
Comment (tpanagos) [inline docs/2/technical/lorawan/lorawan-devices.md:13]: extra capital

File: docs/2/technical/lorawan/lorawan-devices.md [hunk 2]
```diff
@@ -27,23 +27,23 @@ Microshare®'s Smart Facilities solutions are powered by devices that quickly an
...
 
-## 2. Types of Sensors
+## 2. Types of Devices
 ---------------------------------------
 
-### A. Motion Sensors
+### A. Motion Devices
 
-Motion sensors have a variety of applications throughout you environment. Motions sensors have been used to provide desk occupancy, room occupancy, smart parking, predictive cleaning and activity level monitoring solutions. 
+Motion devices have a variety of applications throughout you environment. Motions devices have been used to provide desk occupancy, room occupancy, smart parking, predictive cleaning and activity level monitoring solutions.
```
Comment (tpanagos) [inline docs/2/technical/lorawan/lorawan-devices.md:40]: Motions devices

File: docs/2/technical/lorawan/lorawan-devices.md [hunk 3]
```diff
@@ -27,23 +27,23 @@ Microshare®'s Smart Facilities solutions are powered by devices that quickly an
...
-Motion sensors have a variety of applications throughout you environment. Motions sensors have been used to provide desk occupancy, room occupancy, smart parking, predictive cleaning and activity level monitoring solutions. 
+Motion devices have a variety of applications throughout you environment. Motions devices have been used to provide desk occupancy, room occupancy, smart parking, predictive cleaning and activity level monitoring solutions. 
 
 ### B. Temperature and Humidity Devices
 
-Microshare® works with both indoor and outdoor specific sensors which monitor environmental changes to various locations. Temperature and humidity sensors are applied in indoor air quality monitoring, environment monitoring, fridge monitoring and legionella prevention solutions. 
+Microshare® works with both indoor and outdoor specific devices which monitor environmental changes to various locations. Temperature and humidity devices are applied in indoor air quality monitoring, environment monitoring, fridge monitoring and legionella prevention solutions. 
 
-### C. Feedback Sensors
+### C. Feedback devices
```
Comment (tpanagos) [inline docs/2/technical/lorawan/lorawan-devices.md:46]: Capitalize

File: docs/2/technical/lorawan/lorawan-devices.md [hunk 4]
```diff
@@ -53,22 +53,22 @@ The water leakage solution detects and alerts facilities managers of a potential
...
+Microshare® works with various models of LoRaWAN devices to deliver a versitile platform of data collection and management. 
 
-### A. Browan Sensors
+### A. Browan Devices
 
-Microshare® employs Browan sensors for indoor monitoring solutions.
+Microshare® employs Browan devices for indoor monitoring solutions.
 
-#### Tabs Motion Sensor 
+#### Tabs Motion Sensors
```
Comment (tpanagos) [inline docs/2/technical/lorawan/lorawan-devices.md:71]: Sensors... consistency

File: docs/2/technical/lorawan/lorawan-devices.md [hunk 5]
```diff
@@ -79,49 +79,49 @@ The Tabs motion sensor uses a Passive Infared Detector and Fresnel lens to detec
 
 For more information on the specifications on the Tabs motion sensor and sample decoding code for the device, refer to the [TBMS100 document](/assets/pdf/TBMS100.pdf).
 
-#### Tabs Temperature and Humidity Sensor
+#### Tabs Temperature and Humidity device
```
Comment (tpanagos) [inline docs/2/technical/lorawan/lorawan-devices.md:82]: Caps

File: docs/2/technical/lorawan/lorawan-devices.md [hunk 6]
```diff
@@ -79,49 +79,49 @@ The Tabs motion sensor uses a Passive Infared Detector and Fresnel lens to detec
...
 
 US Model Number: TMHV110-915
 EU Model Number: TMHV110-868
 
-The Tabs temperature and humidity sensor measures the indoor temperature, relative humidity and air quality of your building. 
+The Tabs temperature and humidity sensor device measures the indoor temperature, relative humidity and air quality of your building.
```
Comment (tpanagos) [inline docs/2/technical/lorawan/lorawan-devices.md:87]: device is probably redundant here

File: docs/2/technical/microshare-platform/app-config-examples/trending-air-quality-mcf88.md [hunk 7]
```diff
@@ -17,47 +17,47 @@ The following is JSON example code for a Trending Air Quality MCF88 App.
   "navOptions": [
     {
       "title": "Realtime Desk Availability",
-      "link": "https://dapp.microshare.io/apps/view/5e0f8cd22b0000d1c2607d32"
+      "link": "https://dapp.microshare.io/apps/view/5e0f8cd22b0000d1c260????"
```
Comment (tpanagos) [inline docs/2/technical/microshare-platform/app-config-examples/trending-air-quality-mcf88.md:20]: these may be standard views, so the actual id may matter. consult with lisa

Comment (ColeFeely6) [reply docs/2/technical/microshare-platform/app-config-examples/trending-air-quality-mcf88.md:20]: I'm going to come back to this for the next pr. I talked with Lisa and there's a little bit of changing that needs to be done there.

**Pull request closed without merging**